content/automate/ManageIQ/Transformation/Common.class/__methods__/vmchecktransformed.rb

```diff
@@ -44,8 +44,12 @@ def main
               @handle.set_state_var(:ae_state_progress, 'message' => 'Disks transformation succeeded.', 'percent' => 100)
             end
           rescue => e
-            @handle.set_state_var(:ae_state_progress, 'message' => e.message)
-            raise
+            if @handle.root['ae_state_retries'] > 1
```

Comment:
@fdupont-redhat Are their specific exceptions that we should catch or is it that the first exception is always caught and ignored and along the way we suppress the exception which will never get raised.

Reply:
👍